recipe/meta.yaml

```diff
@@ -55,6 +55,8 @@ requirements:
     - numpy 1.11.*
     - mpi4py 3.0.*  # [mpi != 'nompi']
     - libgdsii >=0.21
+    - ffmpeg
+    - mayavi
```

Comment:
We probably only need these at runtime and not during the build, right?

Reply:
Correct!